File: 7.Multi Language Basics/go/basics.go
package main

import "fmt"

func main() {
    // 1. Output
    fmt.Println("Hello from Go!")

    // 2. Variables (short declaration operator :=)
    score := 85
    average := 78.5

    // 3. Conditionals (no parentheses around condition)
    if score >= 80 {
        fmt.Println("Result: Excellent!")
    } else {
        fmt.Println("Result: Try harder!")
    }

    // 4. Loops (Go only has 'for' loop, no 'while' loop)
    fmt.Print("For Loop output: ")
    for i := 1; i <= 5; i++ {
        fmt.Printf("%d ", i)
    }
    fmt.Println()
    
    // Formatting variables in print
    _ = average // Silence unused variable error in Go compilation
}

File: 7.Multi Language Basics/csharp/basics.cs
using System;

class Program {
    static void Main() {
        // 1. Output
        Console.WriteLine("Hello from C#!");

        // 2. Variables
        int score = 85;
        double average = 78.5;

        // 3. Conditionals
        if (score >= 80) {
            Console.WriteLine("Result: Excellent!");
        } else {
            Console.WriteLine("Result: Try harder!");
        }

        // 4. Loops
        Console.Write("For Loop output: ");
        for (int i = 1; i <= 5; i++) {
            Console.Write(i + " ");
        }
        Console.WriteLine();
    }
}
